### PR TITLE
MDR40-051 Codebase Refinement (Tailwind) -> Navbar

### PR DESCRIPTION
**Pull request by BryanPalad**

### CODE REVIEW

File: src/components/Appbar/Appbar.js [hunk 1]
```diff
@@ -29,89 +26,26 @@ const ApplicationBar = (props) => {
...
-              </Box>
-            </>
-          ) : (
-            <></>
-          )}
-          <AppModal buttonName={"SIGN IN"} buttonColor={'button-black'} title={"ACCOUNT LOGIN"} description={<SystemLogin/>} 
-          buttonStyle={{fontSize: '15px', padding: '10px 25px', borderRadius: '30px', fontWeight: 600}} logo/>
-        </Toolbar>
-      </AppBar>
+      <header className={`${bg ? 'bg-primary h-20' : 'h-24'} flex items-center fixed top-0 w-full text-black dark:text-white z-10 transition-all duration-300`}>
```
Comment (JMSevilla) [inline src/components/Appbar/Appbar.js:29]: As per the changes , did we remove our re-usable application bar here and replace it with header instead ?

Comment (BryanPalad) [reply src/components/Appbar/Appbar.js:29]: Yes, appbar from material ui was removed and changed it into header and className for the implementation of tailwind. Or do you want to revert appbar and modify it's classsname?

Comment (JMSevilla) [reply src/components/Appbar/Appbar.js:29]: If we can much better.

Comment (BryanPalad) [reply src/components/Appbar/Appbar.js:29]: Noted with thanks

**Review by JMSevilla — APPROVED** [review]

**Pull request merged**